edu-blog-vue/components/Metatag.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
I'm not sure if meta-tag is a good name. because meta-tags are usually in the head https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta